src/main/java/frc/robot/commands/SyncSteerOnFly.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.SwerveSubsystem;
+
+public class SyncSteerOnFly extends CommandBase {
+  /** Creates a new SyncSteerOnFly. */
+  public SyncSteerOnFly() {
+    // Use addRequirements() here to declare subsystem dependencies.
```

Comment:
you probably need to add SwerveSubsystem.getInstance() as a requirement.

Reply:
Adding that requirement would unscheduled TeleopSwerve for one roborio cycle, which may be a good thing because this command should not be used the chassis is moving.

I agree with adding the requirement